### PR TITLE
Setting Burnup Peaking Factor to 0.0 when Undefined

### PR DESCRIPTION
**Pull request by wcscherer**
## What is the change?
Originally, the `GlobalFluxInterface` block method `getBurnupPeakingFactor` was set to return `1.0` for the block burnup peaking factor if the factor was not already defined. The factor is changed to return `0.0` if no burnup peaking factor is already set.

## Why is the change being made?
To eliminate confusion around what peaking parameters should be used for downstream analyses, the peaking factor is now set to `0.0` so that if the `burnupPeakingFactor` is undefined, the block burnup peaking factor will be `0.0` and this will propagate to other block properties that use the burnup peaking factor.  Any parameter using this peaking value when `burnupPeakingFactor` is undefined will return `0.0` so that it is clear to not use the results of the parameter. 

---

## Checklist

- [x] This PR has only one purpose or idea.
- [x] Tests have been added/updated to verify that the new/changed code works.

- [x] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [x] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [x] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any important changes.
- [x] The documentation is still up-to-date in the `doc` folder.
- [x] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

File: armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py [hunk 1]
```diff
@@ -334,6 +334,14 @@ def test_getBurnupPeakingFactor(self):
         factor = mapper.getBurnupPeakingFactor(b)
         self.assertEqual(factor, 2.5)
 
+    def test_getBurnupPeakingFactorZero(self):
```
Comment (john-science) [inline armi/physics/neutronics/globalFlux/tests/test_globalFluxInterface.py:337]: Thanks for adding the test!

**Review by john-science — APPROVED** [review]

**Comment by wcscherer** [comment]
@john-science, I'm trying to add my TP email to my profile so it recognizes the commit source. The CLA is signed. I apparently exceeded the verification email limit so I have to wait to try again.

File: armi/physics/neutronics/globalFlux/globalFluxInterface.py [hunk 2]
```diff
@@ -749,7 +749,7 @@ def getBurnupPeakingFactor(self, b: Block):
             burnupPeakingFactor = b.p.fluxPeak / b.p.flux
         elif not burnupPeakingFactor:
             # no peak available. Finite difference model?
-            burnupPeakingFactor = 1.0
+            burnupPeakingFactor = 0.0
```
[the remark this answers is not in the document]
Comment (wcscherer) [reply armi/physics/neutronics/globalFlux/globalFluxInterface.py:752]: ok works for me. Ill push the changes right now

**Comment by wcscherer** [comment]
@john-science Had to rebase and force push to reset the email addresses on the commits so they matched my TP email associated with my account. This then satisfies the CLA.

**Review by onufer — APPROVED** [review]

**Pull request merged**